wsgi/odyssy/announcement/models.py

```diff
@@ -0,0 +1,20 @@
+from __future__ import unicode_literals
+
+from django.db import models
+
+
+# Create your models here.
+
+class Announcement(models.Model):
+    """ Model for Announcement
+    key         -- Primary Key
+    initDate    -- Initialization Date
+    finDate     -- Final Date
+    title       -- Title of the Announcement
+    description -- Description of the Announcement
+    """
+    key = models.AutoField(primary_key=True)
+    initDate = models.DateTimeField()
+    finDate = models.DateTimeField()
+    title = models.CharField(max_length=50)
+    description = models.CharField(max_length=500)
```

Comment:
You can create a `__str__` metho,d it's easy to manage objects in the command line.


![objects all not working](https://cloud.githubusercontent.com/assets/25580776/26335586/8fd658fc-3f8a-11e7-96a9-cf46dc958649.jpg)

It'll easy to manage if `Announcement.objects.all()` shows all objects with their title.